packages/cephalon/src/actions/leave-voice.scope.ts

```
@@ -1,41 +1,44 @@
-  policy: PolicyChecker;
-  voice: VoiceAdapter;
+	logger: Logger;
+	policy: PolicyChecker;
+	voice: VoiceAdapter;
 };
 
 // Builder that can work with or without a live Bot (e.g., broker/CLI)
 export async function buildLeaveVoiceScope(ctx?: {
-  bot?: Bot;
+	bot?: Bot;
 }): Promise<LeaveVoiceScope> {
```

ESLint

[error] Parameter should have an immutability of at least "ReadonlyDeep" (actual: "Mutable").

(functional/prefer-immutable-types)